src/tests/snapshot.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
- [ ] Greate work testing the functions of the app, kindly consider also testing the user interaction by using `React Testing Library`